pkg/util/overridemanager/overridemanager.go

```diff
@@ -190,7 +190,23 @@ func (o *overrideManagerImpl) getOverridersFromOverridePolicies(policies []Gener
 			resourceMatchingPolicies = append(resourceMatchingPolicies, policy)
 		}
 	}
-
+	sort.SliceStable(resourceMatchingPolicies, func(i, j int) bool {
+		implicitPriorityI := util.ResourceMatchSelectorsPriority(resource, resourceMatchingPolicies[i].GetOverrideSpec().ResourceSelectors...)
+		if len(resourceMatchingPolicies[i].GetOverrideSpec().ResourceSelectors) == 0 {
```

Comment:
Emmm, how could the `ResourceSelectors` be empty🤔️? I notice that this field has an annotation `+kubebuilder:validation:MinItems=1`. BTW, why considering priority as `PriorityMatchAll` if empty? Could you enlighten me🤣?

Reply:
> how could the ResourceSelectors be empty
For op/cop , yes.


> I notice that this field has an annotation +kubebuilder:validation:MinItems=1

OP/COP does not have this annotation . https://github.com/karmada-io/karmada/blob/master/pkg/apis/policy/v1alpha1/override_types.go#L45-L46

Reply:
>  why considering priority as PriorityMatchAll if empty?

see https://github.com/karmada-io/karmada/issues/2618